.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,38 @@
+# This GitHub Actions workflow contains one job:
+# -- Build MKDocs uses the latest Ubuntu image and Python 3.9 to build
+# the MKDocs website using the latest Poetry version.
+
+name: Build MKDocs
+
+on:
+  push:
+    branches:
+      -main
+jobs:
+  mkdocs:
+    name: MkDocs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Python 3.9
+        uses: actions/setup-python@v2
+        id: setup-python
+        with:
+          python-version: 3.9
+      - name: Install Poetry
+        uses: Gr1N/setup-poetry@v7
```

Comment:
You should use the same Poetry installation action for all your workflows. I personally prefer the `abatilo` action, but this one seems good as well.